TestFoundation/TestFileManager.swift

```diff
@@ -1084,7 +1082,7 @@ class TestFileManager : XCTestCase {
         try testCopy()
     }
     
-#if !DEPLOYMENT_RUNTIME_OBJC // XDG tests require swift-corelibs-foundation
+#if !DEPLOYMENT_RUNTIME_OBJC && !os(Android) // XDG tests require swift-corelibs-foundation
     
     #if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT // These are white box tests for the internals of XDG parsing:
```

Comment:
This isn't set on linux or Android, so none of these XDG tests are compiled or run anyway.

Reply:
You are right, but the reason for adding the two checks is that `NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT` is there because `@testable` is not supported in Linux, while `!os(Android)` is there because Android will probably never support XDG. If we ever solve the first problem, the second doesn’t go away, so having both checks in place make sense to me (otherwise you will have to check each place where `NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT` is being used for “phantom” conditions, which sounds impossible).

I can change it if everyone disagrees.

Reply:
Sounds good to me, I already approved the pull.